src/main/java/com/rultor/agents/daemons/EndsDaemon.java

```diff
@@ -70,14 +73,29 @@
     public static final String HIGHLIGHTS_PREFIX = "RULTOR: ";
 
     /**
+     * The profile to check.
+     */
+    private final transient ProfileDeprecationAware profile;
+
+    /**
      * Ctor.
      */
     public EndsDaemon() {
```

Comment:
@mkordas In other words, IMHO it is not a good practice to remove a public methods and/or constructors between 2 minor versions. If you don't apply the same logic here, please tell me I will remove it.